parking/shared/clients.py

```diff
@@ -97,8 +99,8 @@ def receive(self, message_type: type) -> asyncio.Future:
         future = asyncio.Future()
         if self._message_queue[message_type]:
             future_set_result_unless_cancelled(future, self._message_queue[message_type].popleft())
-            logger.info("message recieved as expected: '{}'".format(message_type))
```

Comment:
Maybe it would be better to demote these to logger.debug or something rather than outright removing them?

Especially for the success log.